versioned_docs/version-0.9/Installation/updating.md

```diff
@@ -77,12 +77,50 @@ sudo chown -R nginx:nginx /var/www/controlpanel/
 sudo chown -R apache:apache /var/www/controlpanel/
 ```
 
-### Restarting Queue Workers
+### Updating Queue Workers
+
+Run the following commands to edit the systemd file:
+
+```bash
+cd /etc/systemd/system
+nano ctrlpanel.service
+```
+
+Delete the content and paste in this:
+
+```bash
+# Ctrlpanel Queue Worker File
+# ----------------------------------
+
+[Unit]
+Description=Ctrlpanel Queue Worker
+
+[Service]
+# On some systems the user and group might be different.
+# Some systems use `apache` or `nginx` as the user and group.
+User=www-data
+Group=www-data
+Restart=always
+ExecStart=/usr/bin/php /var/www/controlpanel/artisan queue:work --sleep=3 --tries=3
```

Comment:
This may not use the php version that you want.
You can check with /usr/bin/php -v
It depends on distro / php setup but it could look like this:
ExecStart=/usr/bin/php8.3 /var/www/controlpanel/artisan queue:work --sleep=3 --tries=3

Reply:
hm... if i were to add a command before that which would help the user figure out if the php folder is /usr/bin/php or /usr/bin/php8.3 would this solve the issue?

Reply:
Sure

Reply:
alright i'll look into it.

Reply:
the new commit solves this issue.